docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -180,14 +180,14 @@ Use the {product-long-api-designer} web console to create an Apache Avro, JSON S
 
 ifndef::qs[]
 .Prerequisites
-* You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
+* You're logged in to the {product-long-api-designer} web console at {service-url-api-designer}[^].
 endif::[]
 
 .Procedure
-. In the {service-url-api-designer}[{product-api-designer} web console], click *Create a schema or API design*.
+. In the {service-url-api-designer}[{product-long-api-designer} web console^], click *Create a schema or API design*.
 . Complete the form to provide the details of the new design:
 .. *Name*: Enter a unique design name, such as `my-schema-design`.
-.. *Summary*: Optional: Enter a short description of the design.
+.. *Summary*: _Optional:_ Enter a short description of the design.
 .. *Type*: From the list, select *Apache Avro*, or select another schema type.
```

Comment:
Should Apache Avro be a literal?

Reply:
I don't think so -- it's an item that the user selects from a list, not text that the user enters in a text field.